ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -61,11 +65,11 @@ Create a transition to define the desired business process.
 
 #### Prerequisites
 
-The basic settings of the workflow have been defined and the workflow editor is displayed, see [Define the workflow basic settings](#define-the-workflow-basic-settings).
+The basic settings of the workflow have been defined, and the workflow editor is displayed, see [Define the workflow basic settings](#define-the-workflow-basic-settings).
```

Comment:
Ggf. diese Korrektur durchgängig übernehmen (s. z.B. Prerequisites in Connect the transation), wobei ich finde da braucht man kein Komma (nur zwei Elemente in der Aufzählung)

Reply:
Da braucht man kein komma